module-2/backend/week_two.md

```diff
@@ -23,22 +37,35 @@ class Team << ActiveRecord::Base
   belongs_to :owner
 end
 ```
-
 Now how would you find the owner of the team with an id of 4?
+team.owner.name
```

Comment:
👍 This would find the owner's name. We're technically just looking for the owner object, but it looks like you've got the right idea.